docs/setup_zms.md

```diff
@@ -52,7 +52,7 @@ $ bin/setup_dev_zms.sh
 
 Running this setup script completes the following two tasks:
 
-* Generate a unique private key that ZMS Server will use to sign any NTokens it issues
+* Generate a unique private key that ZMS Server will use to sign any X509 certificates it issues
```

Comment:
this is not correct since ZMS Server does not sign any x.509 certs. the previous text is correct. it is used for singing Principal Tokens (NTokens)